src/wp-includes/style-engine/class-wp-style-engine.php

```diff
@@ -305,6 +305,12 @@ final class WP_Style_Engine {
 				),
 				'path'          => array( 'typography', 'letterSpacing' ),
 			),
+			'writingMode'    => array(
+				'property_keys' => array(
+					'default' => 'writing-mode',
+				),
+				'path'          => array( 'typography', 'writingMode' ),
+			),
```

Comment:
Should this function have an `@since` comment about this addition? I know this backport is sort of a bug fix but at some point, the docblock would've normally had one stating the enhancement/new feature, right?

Reply:
Oh, good call! Yes for completeness sake, I'll add in a `@since` for this one 👍